Dynatrace/dynatrace-operator
# Feat/add ff for oa livenessprobe

## Description

Currently it can happen, that the agent is not getting ready in time (20 mins) and therefore is being killed by the livenessprobe, as the installer is still running. For such situations to still allow agent startup, we should introduce a feature flag to skip the probe.

## How can this be tested?

Create a dynakube with the feature flag `feature. dynatrace. com/oneagnet-skip-liveness-probe` set to true, rollout an agent daemonset, check if the daemonset has no livenessprobe configured
